DataHub/UserInterface/02a_Attributes.md

```diff
@@ -0,0 +1,259 @@
+# Attributes
+
+*DataHub > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/Attributes/Attributes.png "[Attributes]")
+
+**Attribute list**
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+  Click this button to display the search bar and search for an attribute.
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+  Click this button to update the list of attributes.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+  Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. You cannot select multiple checkboxes at once.
+
+- [EDIT]   
+  Click this button to edit the selected attribute. This button is only displayed if the checkbox of an attribute is selected. Alternatively, you can click directly a row in the list to edit an attribute.   
+  For detailed information, see [Edit an attribute](../Integration/01_ManageAttributes.md#edit-an-attribute).
+
+- [DELETE]   
+  Click this button to delete the selected attribute. This button is only displayed if the checkbox of an attribute is selected.       
+  For detailed information, see [Delete an attribute](../Integration/01_ManageAttributes.md#delete-an-attribute).
+
+The list displays all attributes. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+- *Name*   
+  Attribute name.
+
+
+- *Status*   
+  Attribute status. The following statuses are available:
+  - ![Status](../../Assets/Icons/Status01.png "[Status]") **Active**
+  - ![Status](../../Assets/Icons/Status04.png "[Status]") **Inactive**   
+
+
+- *Group*   
+  Attribute group assigned to the attribute. If the attribute is not assigned to any attribute group, *not assigned* is displayed.
+
```

Comment:
Attribute group assigned to the attribute oder Attribute group to which the attribute is assigned? 
Oder ist es das gleiche?